### PR TITLE
Init whole project alpha 0.001

### PR DESCRIPTION
**Pull request by Duan-JM**
1. move old sw_project to `jarvice/module/legacy`.
2. create README for `jarvis`. (WIP)
3. create `service.py` and `test`. (WIP)

**additional**

1. module in `process.py` and `action.py` normally functional. `input.py` remain some bugs to fix, (hang out for HW to give demo)
2. service script in `jarvis/service` folder is still under development.

### CODE REVIEW

File: conference_system/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<h1 id="conference" align="center">ConferenceSystem</h1>
+
+<p align="center">
```
Comment (huan) [inline conference_system/README.md]: Can we use Markdown instead of HTML for our documentation?

File: conference_system/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    │   │       ├── Misc
+    │   │       ├── PascalVOC-Detection
+    │   │       └── quick_schedules
+    │   ├── __init__.py
+    │   ├── input.py
+    │   ├── process.py
+    │   ├── README.md
+    │   └── utils.py
+    └── service
+        └── lecture_generate_service.py
```
Comment (huan) [inline conference_system/README.md]: Please fix the layout problem from here.

**Review by huan — CHANGES_REQUESTED** [review]
It is great to see our codebase is growing!

File: conference_system/README.md (hunk 2 above)
Comment (Duan-JM) [reply conference_system/README.md]: Thx for point out the issue. Dir tree have fixed (reference to the commit 50787d5)

File: conference_system/README.md (hunk 1 above)
Comment (Duan-JM) [reply conference_system/README.md]: Using markdown is good to me. Doc is still under developing, so I believe we can hang this pr until this weekend. Till then, we will get a much better doc to merge.

**Comment by huan** [comment]
Ok. Please let me know when you are ready for merge.

**Comment by Duan-JM** [comment]
After discussion with the @ximingxing, as far as we considered, instead of using markdown for the whole README, it could be great to contain few HTML to centre the title and add `tag` in the doc.

Basically, the code in legacy is good to go and well documented and could be reused in the future.
Code in the `input.py` and `action` immigrate from the legacy, and code in the `process.py` is well tested and is well documented in the code. 

Maybe code can be merged, for now, and to be modified later?

**Review by huan — COMMENTED** [review]
Of course, let's merge it first if you believe it's ready to go!

Thanks for your great effort to make the magic mirror project better!

**Pull request closed without merging**